lib/reference/properties.mjs

```diff
@@ -618,3 +618,50 @@ export const longhandTimeProperties = new Set([
 ]);
 
 export const shorthandTimeProperties = new Set(['transition', 'animation']);
+
+export const legacyProperties = new Set([
```

Comment:
[ask] Is it possible to add them to `known-css-properties` upstream?

Reply:
It's possible but, for the same reasons that `css-functions-list` or `csstree` can be slow to release, we want to fix it on our side first.
Also, in the same way that `html-tags` doesn't include deprecated elements, we run the risk that their package policy would be against adding legacy properties.
i.e. we should open a PR there but we should still fix it in the meantime.

Reply:
known-css/known-css-properties#174

Reply:
Thanks for opening the PR upstream. It's okay to fix them on our side for now. 👍🏼 

In addition, can you add a TODO comment with the PR link so as not to forget to remove duplicate properties when `known-CSS-properties` releases a new version in the future?

Reply:
```suggestion
/** @see known-css/known-css-properties#174 */
const legacyProperties = new Set([
```

Reply:
```suggestion
/** @see known-css/known-css-properties#174 */
export const legacyProperties = new Set([
```

Reply:
https://github.com/known-css/known-css-properties/pull/174#issuecomment-2176842525

Reply:
Opened #7771. Can you take a look? Thanks.

Reply:
I tried to commit and got:

```console
stylelint $ git push --no-verify
remote: Permission to stylelint/stylelint.git denied to Mouvedia.
fatal: impossible d'accéder à 'https://github.com/stylelint/stylelint.git/' : The requested URL returned error: 403
```

We had the same problem on #7713.

Reply:
Um, weird. You now have the *Write* role in this repo... 🤔

Reply:
Um..., I think you can push commits like this https://github.com/stylelint/stylelint/commit/f4e8b63cdcb097d3855f07dfe487bada104c50a6

Reply:
Really? I must be doing something wrong then.
I thought I needed the aforementioned fined-grained token to be accepted to push on `dependabot/*` branches.
I am not really in the mood to investigate right now; Ill let you do the `legacyProperties` removal.

Reply:
Ah, I see. It seems that Dependabot branches are especially protected. Alright, I'll push later. 👌🏼